### PR TITLE
Adds Restify and Bunyan externs

### PR DESCRIPTION
**Pull request by JbIPS**

### CODE REVIEW

File: js/npm/restify/Error.hx [hunk 1]
```diff
@@ -0,0 +1,431 @@
+package js.npm.restify;
+
+typedef ErrorOptions = {
+    ?message : Dynamic,
+    ?statusCode : Int
+}
+
+@:native('errors.HttpError')
+extern class HttpError extends js.Error implements js.extern.Extern<'restify'>
```
Comment (clemos) [inline js/npm/restify/Error.hx]: `js.extern.Extern` ?

Comment (JbIPS) [inline js/npm/restify/Error.hx]: Useless, thus removed :)

Comment (clemos) [inline js/npm/restify/Error.hx]: I'm not sure about the `@:native('errors.HttpError')` either...
What is it supposed to do ?

Comment (JbIPS) [inline js/npm/restify/Error.hx]: I thought it was a standard meta linking to a JS name/class. But finally it looks like it was specific to the extern system I port it from. Will remove too

**Pull request merged**